kokoro/ubuntu/periodic.sh

```diff
@@ -13,5 +13,25 @@ mvn -B clean install
 # https://stackoverflow.com/questions/3459928/running-a-specific-maven-plugin-goal-from-the-command-line-in-a-sub-module-of-a/26448447#26448447
 # https://stackoverflow.com/questions/11091311/maven-execjava-goal-on-a-multi-module-project
 cd dashboard
+
+# Step 1: Generate dashboards for the released BOMs with the latest DashboardMain class
+# For example: https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/1.1.1/index.html
+
+# Querying https://search.maven.org/classic/#api with
+# groupId:com.google.cloud AND artifactId:libraries-bom, receiving latest (sorted) 1000 versions.
+# Note: if `rows` parameter is omitted, it only returns latest 10 items.
+SONATYPE_RESPONSE=`curl 'https://search.maven.org/solrsearch/select?q=g:%22com.google.cloud%22+AND+a:%22libraries-bom%22&core=gav&rows=1000&wt=json'`
+# Example: '1.0.0 1.1.0'
+VERSIONS=`echo $SONATYPE_RESPONSE | perl -nle 'print $1 while m/"v":"(.+?)"/g'`
```

Comment:
How reliable is parsing JSON with regexps? What if Sonatype reformats? How are errors reported?